src/edwards.rs

```diff
@@ -104,6 +104,13 @@ use core::ops::{Mul, MulAssign};
 #[cfg(feature = "digest")]
 use digest::{generic_array::typenum::U64, Digest};
 
+#[cfg(feature = "group")]
+use {
+    group_crate::{cofactor::CofactorGroup, prime::PrimeGroup, GroupEncoding},
```

Comment:
If you add `extern crate group_crate as group`, you can `use group::` for imports.

It can have a `TODO` comment to remove that directive after a migration to namespaced features (which we might consider prior to a final 4.0 release, especially as it would be helpful for #481 as well)